### PR TITLE
[topics] use a screen for topics rather than search screen

### PR DESCRIPTION
**Pull request by haileyok**

### CODE REVIEW

File: src/screens/Topic.tsx [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+    isFetched,
+    isFetchingNextPage,
+    isLoading,
+    isError,
+    error,
+    refetch,
+    fetchNextPage,
+    hasNextPage,
+  } = useSearchPostsQuery({
+    query: topic.replaceAll(' ', '+').replaceAll('%20', '+'),
```
Comment (estrattonbailey) [inline src/screens/Topic.tsx:150]: Do we need to decode here first too? Haven't tested just stood out to me

Comment (haileyok) [reply src/screens/Topic.tsx:150]: oh yea lets do that

**Review by estrattonbailey — APPROVED** [review]
Nice thanks! Little behind base but should be fine to merge in right?

**Comment by haileyok** [comment]
made another one because this was missing a push or something, but same thing otherwise.

**Pull request closed without merging**